ReSharper.FSharp/.idea/.idea.ReSharper.FSharp/.idea/runConfigurations/Rider_Debug__Unix_.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Rider Debug (Unix)" type="RunExe" factoryName=".NET Executable">
```

Comment:
With the following change I'm able to launch the IDE but something is very off with it: daemon doesn't work, there're no exceptions, everything just doesn't seem to be working properly.
```suggestion
  <configuration default="false" name="Rider Debug (Unix)" type="RunNativeExe" factoryName="Native Executable">
```
<configuration default="false" name="Rider Debug (Unix)" type="RunNativeExe" factoryName="Native Executable">

Reply:
OK, with the latest Rider dev build debugging does work, but it still requires the change above. @ForNeVeR @citizenmatt Is it expected?

Reply:
Yes. The .NET Executable run configuration on Mac will try to launch the `gradlew` shell script as a Mono executable, and so you get the `File does not contain a valid CIL image` error. You need to use the `RunNativeExe` config to launch a shell script, and it's what we're doing in the product, too.

Reply:
Thanks, guys. I wasn't aware of that issue with Mono (probably this is something that will be improved in 2021.2, though).